c++/src/kj/main.c++

```diff
@@ -245,12 +245,7 @@ struct MainBuilder::Impl {
 
   struct CharArrayCompare {
```

Comment:
You can just delete CharArrayCompare altogether, and remove the type parameter from `std::map` below, since comparing using `operator<` is the default.